server.go

```diff
@@ -1500,6 +1506,13 @@ func (s *Server) ReadSeries(database, retentionPolicy, name string, tags map[str
 // Returns a resultset for each statement in the query.
 // Stops on first execution error that occurs.
 func (s *Server) ExecuteQuery(q *influxql.Query, database string, user *User) Results {
+	// Authorize user to execute the query.
+	if err := Authorize(user, q, database); err != nil {
```

Comment:
Can  you give me an example of what the JSON body returned would be like if the user is authorized to execute the first statement, but not the second?